src/main/java/com/vispana/api/model/apppackage/Content.java

```diff
@@ -0,0 +1,17 @@
+package com.vispana.api.model.apppackage;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import java.util.List;
+
+public class Content {
+
+  @JacksonXmlProperty(isAttribute = true)
+  private String id;
+
+  @JacksonXmlProperty(localName = "group")
+  private TopGroup topGroup;
```

Comment:
Not super important right now, but it would be nice to eventually support [flat distributions](https://docs.vespa.ai/en/performance/sizing-search.html#flat-content-distribution) as well.

Reply:
Can add a separate PR for it.